ipykernel/comm/comm.py

```diff
@@ -21,15 +21,22 @@ class Comm(LoggingConfigurable):
     shell = Instance('IPython.core.interactiveshell.InteractiveShellABC',
                      allow_none=True)
     kernel = Instance('ipykernel.kernelbase.Kernel')
-    def _kernel_default(self):
+
+    @default('kernel')
+    def _default_kernel(self):
```

Comment:
Yeah, I found this when doing the update for JupyterHub. For more advanced uses, the changes are really nice, but for most of our simple use it really is not an improvement. Using traitlets for simple config has become more tedious than it used to be.